packages/next/taskfile.js

```diff
@@ -61,6 +61,12 @@ const externals = {
 
   'terser-webpack-plugin':
     'next/dist/build/webpack/plugins/terser-webpack-plugin',
+
+  /**
+   * TODO: add @swc/helpers as once ncc replaces ts-loader
+   * with swc-loader and externalHelpers becomes an option:
+   * https://github.com/vercel/ncc/pull/926/
+   */
```

Comment:
Note: Currently in https://github.com/vercel/ncc/pull/926/, `externalHelpers` is always disabled.